woocommerce/woocommerce
# Suppress second scrollbar on WooCommerce > Extensions pages

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Initially for review by @woocommerce/alpha-fire before consideration for inclusion in `trunk`. Closes [WCCOM #18029](https://github.com/Automattic/woocommerce.com/issues/18029).

Adjusts CSS to prevent "double-scrollbars" appearing on WooCommerce > Extensions pages on computers configured to "always show scrollbars". See video for demonstration of the bug and how to replicate it:

https://github.com/woocommerce/woocommerce/assets/53293/8a057495-0ce7-47cd-bc65-a1109760ecd2

(This is necessary because the layout of WooCommerce > Extensions pages adjusts the page height (to compensate for the larger header). On systems with scrollbars set to always-visible, this results in a ~1px overflow of the inner content and double-scrollbars are visible. This CSS mitigates that.)

### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Check out this branch to your environment and compile the WC Admin assets, e.g. with `cd plugins/woocommerce-admin && pnpm i && pnpm start:hot`
2. Configure your operating system to "always show scrollbars"
3. View a WooCommerce > Extensions page and observe that only a single scrollbar appears.
4. Please test around this change.

### Changelog entry




<details>

#### Significance




#### Type




#### Message

#### Comment

Fixed a bug that caused redundant extra scrollbars to appear on WooCommerce > Extensions pages for some browser/OS/configuration combinations.

</details>
